src/qt/transactionrecord.cpp

```diff
@@ -18,6 +18,7 @@
 #include <utilmoneystr.h>
 
 #include <stdint.h>
+#include <QDebug>
```

Comment:
what is this use for?

Reply:
Fixed - this was left over from when I had debug logging.